webrender/src/device.rs

```diff
@@ -2215,18 +2307,42 @@ impl<B: hal::Backend> Device<B, hal::Graphics> {
         texture.filter = filter;
         texture.layer_count = layer_count;
         texture.render_target = render_target;
+        //println!("texture.width={:?}", texture.width);
```

Comment:
I recommend to use the `log` macros. It's already hooked up, all you need is to use `debug!`,`warn!`,`info!` macros and become familiar with how to see it (e.g. `RUST_LOG=webrender=debug cargo run ...`)